stellargraph/layer/gcn.py

```diff
@@ -143,8 +149,8 @@ def build(self, input_shapes):
             input_shapes (list of int): shapes of the layer's inputs (node features and adjacency matrix)
 
         """
-        feat_shape = input_shapes[0]
-        input_dim = int(feat_shape[-1])
+        feature_shape, *As_shapes = input_shapes
```

Comment:
Doesn't seem like `As_shapes` is used in this method, should we just leave it as `feature_shape = input_shapes[0]`?